### DISCUSSION TITLE
Pull logs from seq instance using Golang

### DISCUSSION THREAD

**Discussion by Asiya-Yunusa** [datalust/seq-tickets#1988] [Help]
Hello everyone, I'll like to know if it possible to pull logs or data of resources from seqs storage?

**Comment by liammclennan** [comment]
Hi @Asiya-Yunusa 

I'm not sure what you are asking for. Can you please provide some more detail about what you would like to retrieve from Seq?

Logs can be retrieved via the [Seq API](https://docs.datalust.co/docs/using-the-http-api), or using the [Seq CLI](https://docs.datalust.co/docs/command-line-client).

**Reply by Asiya-Yunusa** [reply]
Hello @liammclennan yes I have been able to get the events api for the seq instance. I am trying to get the data out of Seq's storage.

**Reply by liammclennan** [reply]
If you want to retrieve all of the log data try https://docs.datalust.co/docs/command-line-client#search

Another options is https://github.com/datalust/seq-app-jsonarchive
